engine/src/main/java/com/sandboni/core/engine/finder/bcel/visitors/ClassUtils.java

```diff
@@ -15,6 +15,12 @@
     // Is safe to have this object living in parallel execution because classpath objects are different
     // for each project and can be accessed/reused during build lifecycle
     private static final ConcurrentMap<String, ClassPath> classPathMap = new ConcurrentHashMap<>();
+    private static final HashSet<String> reflectionClassSearchPatters = new HashSet<>(Arrays.asList(
```

Comment:
better use Enum

Reply:
ok.

Reply:
1. centralized place where we can update new reflection packages.
2. what do you do in your tests? duplicate the strings?

Reply:
1. it's centralized in both cases
2. N/A because I have an actual import statement
I'll change it but I still like to understand why it's better.